src/Sarif.Converters/ConverterFactory.cs

```diff
@@ -1,131 +1,32 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
```

Comment:
This class formerly combined the logic for plugins and built-in converters. We factor the logic into two new classes, `PluginConverterFactory` and `BuiltinConverterFactory`. The `ConverterFactory` class remains behind as an abstraction that implements the Chain of Responsibility. See the right-hand version; treat is as new code.